WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -412,7 +413,7 @@ extension WebKitViewController: WKUIDelegate {
         }
 
         if !ReachabilityUtils.isInternetReachable() {
-            ReachabilityUtils.showAlertNoInternetConnection()
+            ReachabilityUtils.showNoInternetConnectionNotice(message: ReachabilityUtils.noConnectionMessage())
```

Comment:
With the new change in `showNoInternetConnectionNotice`, the `message` should no longer be needed.